app/src/main/java/com/kickstarter/ui/activities/CreatorDashboardActivity.java

```diff
@@ -1,69 +1,82 @@
 package com.kickstarter.ui.activities;
 
-import android.content.Intent;
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
+import android.support.design.widget.BottomSheetBehavior;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.util.Pair;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
-import android.util.Pair;
 
 import com.kickstarter.R;
 import com.kickstarter.libs.BaseActivity;
-import com.kickstarter.libs.RefTag;
 import com.kickstarter.libs.qualifiers.RequiresActivityViewModel;
 import com.kickstarter.models.Project;
 import com.kickstarter.services.apiresponses.ProjectStatsEnvelope;
-import com.kickstarter.ui.IntentKey;
-import com.kickstarter.ui.adapters.CreatorDashboardAdapter;
+import com.kickstarter.ui.adapters.CreatorDashboardBottomSheetAdapter;
+import com.kickstarter.ui.fragments.CreatorDashboardFragment;
 import com.kickstarter.viewmodels.CreatorDashboardViewModel;
 
+import java.util.List;
+
 import butterknife.Bind;
 import butterknife.ButterKnife;
 
 import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
 
 @RequiresActivityViewModel(CreatorDashboardViewModel.ViewModel.class)
 public final class CreatorDashboardActivity extends BaseActivity<CreatorDashboardViewModel.ViewModel> {
-  private CreatorDashboardAdapter adapter;
 
-  protected @Bind(R.id.creator_dashboard_recycler_view) RecyclerView creatorDashboardRecyclerView;
+  private CreatorDashboardBottomSheetAdapter bottomSheetAdapter;
+  private BottomSheetBehavior bottomSheetBehavior;
+
+  protected @Bind(R.id.creator_dashboard_bottom_sheet_recycler_view) RecyclerView bottomSheetRecyclerView;
 
   @Override
   protected void onCreate(final @Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.creator_dashboard_layout);
     ButterKnife.bind(this);
 
-    this.adapter = new CreatorDashboardAdapter();
-    this.creatorDashboardRecyclerView.setAdapter(this.adapter);
-    this.creatorDashboardRecyclerView.setLayoutManager(new LinearLayoutManager(this));
+    // Set up the bottom sheet recycler view.
+    this.bottomSheetAdapter = new CreatorDashboardBottomSheetAdapter();
+    this.bottomSheetRecyclerView.setAdapter(this.bottomSheetAdapter);
+    this.bottomSheetRecyclerView.setLayoutManager(new LinearLayoutManager(this)); // todo: reuse LayoutManager?
```

Comment:
ooo sry you can delete this todo now, since we only have one LinearLayoutManager instance now in this file